napi-inl.h

```diff
@@ -392,6 +425,38 @@ inline std::u16string String::Utf16Value() const {
 // Object class
 ////////////////////////////////////////////////////////////////////////////////
 
+namespace helper {
+
+template <typename Key>
+class PropertyLValue {
+public:
+  operator Value() const;
+  PropertyLValue& operator=(Value value);
+private:
+  PropertyLValue(Object object, Key key);
+  Object _object;
+  Key _key;
+  
+  friend class Napi::Object;
+};
+
+template <typename Key>
+inline PropertyLValue<Key>::operator Value() const {
+  return _object.Get(_key);
+}
+
+template <typename Key>
+inline PropertyLValue<Key>& PropertyLValue<Key>::operator =(Value value) {
```

Comment:
@kkoopa Ah, thanks, updated (with `operator =`, which seems to be the prevalent style for declarations/definitions here).